src/Microsoft.Identity.Client/Internal/Cache/AccessTokenCacheItem.cs

```diff
@@ -35,6 +35,10 @@ namespace Microsoft.Identity.Client.Internal.Cache
     [DataContract]
     internal class AccessTokenCacheItem : BaseTokenCacheItem
     {
+        protected AccessTokenCacheItem(RequestContext requestContext)
```

Comment:
you should not pass request context to classes that do not require logging. This class and User class are a great example of that.